ga4gh/datamodel/datasets.py

```diff
@@ -227,6 +226,27 @@ def __init__(self, localId, dataDir, backend):
                     self, localId, bamPath, backend)
                 self.addReadGroupSet(readGroupSet)
 
+    def descriptionExists(self, fname):
+        # check for existence of description JSON
+        return os.path.exists(fname)
+
+    def parseDescriptionJson(self, str):
+        # parse into python dict
+        return json.loads(str)
+
+    def descriptionFilename(self, path, localId):
```

Comment:
We'd usually use ``getDescriptionFilename`` here, to make it clearer that your returning the filename.